lonestar/mining/gpu/kcl/kcl_gpu.cpp

```diff
@@ -1,18 +1,37 @@
 // Copyright 2019, University of Texas at Austin
 // Authors: Xuhao Chen <cxh@utexas.edu>
+#include "galois/Galois.h"
 #include "kcl.h"
 #include "lonestarmine.h"
+#include "llvm/Support/CommandLine.h"
```

Comment:
Not sure what the interaction with llvm and GPU code is, though this is the CPP end so it's probably fine.